src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -83,10 +83,10 @@ public function new(Request $request): Response
     }
 
 <?php if ($use_attributes) { ?>
-    #[Route('/{<?= $entity_identifier ?>}', name: '<?= $route_name ?>_show', methods: ['GET'])]
+    #[Route('/{<?= $entity_identifier ?>}', name: '<?= $route_name ?>_show', methods: ['GET'], requirements: ['<?= $entity_identifier ?>' => '\d+'])]
```

Comment:
Howdy, thanks for the work on this. My biggest concern with this is if my entity has a string based id, e.g. uuid's, these routes will no longer work. For us to properly implement this dynamically, we would need to know the scalar type of the  `id` for the entity.

Reply:
Should I make a better regex to fix that?

Reply:
Any news? @jrushlow